wordpress-mobile/gutenberg-mobile
# Fix Bold, Italic, Strikethrough format

Bold, Italic and Strikethrough format were not working properly because of some changes made on RichText web component.

To test:
 - Start the demo app
 - Add a paragraph block
 - Press on bold, italic, and strikethrough format buttons
 - Check if no red screen appears
 

PR submission checklist:
